Assignment2/Q1.java

```diff
@@ -0,0 +1,148 @@
+//The tree created is a binary tree to match the example given but the method does not make any use of binary tree features 
+
+class Q1
+{
+	public static void main(String[] args)
+	{
+		BinaryTree tree = new BinaryTree(16);
+		tree.add(9);
+		tree.add(18);
+		tree.add(19);
+		tree.add(3);
+		tree.add(14);
+		tree.add(1);
+		tree.add(5);
+
+		System.out.println("OK");
+
+		find(tree, 5);
+	}
+
+	public static void find(BinaryTree tree, int valueToFind)
+	{
+		if (tree.root.value == valueToFind)
+		{
+			System.out.println(valueToFind + " is the root - it has no ancestors");
+		}
+		else if(findHelp(tree.root, valueToFind) == 0)
+		{
+			System.out.println(valueToFind + " is not part of the tree");
+		}
+	}
+
+	public static int findHelp(Node consider, int valueToFind)
```

Comment:
You seem to be using this to return a boolean (1 for true, 0 for false), so why have you declared this as returning an integer. Maybe you could add some javadoc to explain the intention of your method?

Nit: as this is an inner helper method it could be private.